### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Addresses https://github.com/dotnet/AspNetCore.Docs/issues/16950

[Internal review URL](https://review.docs.microsoft.com/en-us/dotnet/core/about?branch=pr-en-us-17131#frameworks)

### CODE REVIEW

File: docs/core/about.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This integration is provided, in part, by the contributors of the [OmniSharp](ht
 
 Multiple frameworks have been built on top of .NET Core:
 
-- [ASP.NET Core](/aspnet/core/)
+- [ASP.NET Core](/aspnet/core/), which provides a framework for building modern cloud-based internet connected applications, such as web apps, IoT apps, and mobile backends.
```
Comment (mairaw) [inline docs/core/about.md]: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/back-end

```suggestion
- [ASP.NET Core](/aspnet/core/), which provides a framework for building modern cloud-based internet connected applications, such as web apps, IoT apps, and mobile back ends.
```

File: docs/core/about.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ This integration is provided, in part, by the contributors of the [OmniSharp](ht
 
 Multiple frameworks have been built on top of .NET Core:
 
-- [ASP.NET Core](/aspnet/core/)
+- [ASP.NET Core](/aspnet/core/), which provides a framework for building modern cloud-based internet connected applications, such as web apps, IoT apps, and mobile back ends.
```
Comment (tdykstra) [inline docs/core/about.md]:
```suggestion
- [ASP.NET Core](/aspnet/core/), a framework for building modern cloud-based internet connected applications, such as web apps, IoT apps, and mobile back ends.
```

**Review by tdykstra — COMMENTED** [review]
If we add a description for ASP.NET, we should add descriptions for the other two as well.

**Comment by Rick-Anderson** [comment]
> If we add a description for ASP.NET, we should add descriptions for the other two as well.

Can you do that? I couldn't find one to copy.

**Comment by tdykstra** [comment]
I recommend we close this in favor of https://github.com/dotnet/docs/pull/16680

**Comment by Rick-Anderson** [comment]
@tdykstra  I'll wait until #16680 merges. I've been tasked to highlight ASP.NET Core and #16680  does the opposite, removes it.

**Pull request closed without merging**